artifactory/v1/repositories.go

```diff
@@ -207,7 +201,6 @@ type RemoteRepository struct {
 	MaxUniqueTags                     *int                    `json:"maxUniqueTags,omitempty"`
 	MismatchingMimeTypesOverrideList  *string                 `json:"mismatchingMimeTypesOverrideList,omitempty"`
 	MissedRetrievalCachePeriodSecs    *int                    `json:"missedRetrievalCachePeriodSecs,omitempty"`
-	Nuget                             *Nuget                  `json:"nuget,omitempty"`
```

Comment:
I think we should put this back with a comment regarding that it is deprecated since 6.9. Then we can remove it later. I would group it with the new fields down the bottom and note they are mutually exclusive.